src/game/LFG/LFGMgr.h

```diff
@@ -0,0 +1,138 @@
+/*
+ * This file is part of the CMaNGOS Project. See AUTHORS file for Copyright information
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+ */
+
+#ifndef MANGOSSERVER_LFGMGR_H
+#define MANGOSSERVER_LFGMGR_H
+
+#include <list>
+#include <map>
+
+#include "Policies/Singleton.h"
+#include "Common.h"
+
+enum ClassRoles
+{
+    LFG_ROLE_NONE   = 0x00,
+    LFG_ROLE_TANK   = 0x01,
+    LFG_ROLE_HEALER = 0x02,
+    LFG_ROLE_DPS    = 0x04
+};
+
+enum RolesPriority
+{
+    LFG_PRIORITY_NONE   = 0,
+    LFG_PRIORITY_LOW    = 1,
+    LFG_PRIORITY_NORMAL = 2,
+    LFG_PRIORITY_HIGH   = 3
+};
+
+enum PlayerLeaveMethod
+{
+    PLAYER_CLIENT_LEAVE = 0,
+    PLAYER_SYSTEM_LEAVE = 1
+};
+
+enum GroupLeaveMethod
+{
+    GROUP_CLIENT_LEAVE  = 0,
+    GROUP_SYSTEM_LEAVE  = 1
+};
+
+#define MAX_DPS_COUNT = 3
```

Comment:
Please for small bit of unification, do the enum like here: https://github.com/cmangos/mangos-wotlk/blob/master/src/game/LFG/LFGDefines.h#L157

So that there is at least a shred of unification for LFG across cores in cmangos. (cos rn there isnt :D )

Reply:
I'm sorry, do you mean this one only or all enums in lfg code?

Reply:
This one only, the rest would be too much of a pain to impose on you.